server/routes.js

```diff
@@ -0,0 +1,18 @@
+var Database = require ('./database/config.js');
+
+module.exports = function(app){
```

Comment:
Instead of doing this, use this method:

`var router = require('express').Router();`

Then when you are calling post
`router.route('/api/addnewmessage').post(function(req, res){`
at the end of the file, have 
`module.exports = router;`